tanner/dorks_manager.py

```diff
@@ -31,25 +31,25 @@ def __init__(self):
                 dorks = dorks.split()
             if isinstance(dorks, set):
                 dorks = [x for x in dorks if x is not None]
-            await redis_client.sadd(redis_key, dorks)
+            await redis_client.sadd(redis_key, *dorks)
 
     async def extract_path(self, path, redis_client):
         extracted = re.match(patterns.QUERY, path)
         if extracted:
             extracted = extracted.group(0)
             try:
-                await redis_client.sadd(self.user_dorks_key, [extracted])
-            except asyncio_redis.NotConnectedError as connection_error:
+                await redis_client.sadd(self.user_dorks_key, *[extracted])
+            except aioredis.ProtocolError as connection_error:
                 self.logger.error('Problem with redis connection: %s', connection_error)
 
     async def init_dorks(self, redis_client):
         try:
-            transaction = await redis_client.multi()
+            transaction = redis_client.multi_exec()
             dorks_exist = await transaction.exists(self.dorks_key)
```

Comment:
Plase look at the warning here 
https://aioredis.readthedocs.io/en/v1.1.0/start.html?highlight=multi_exec#multi-exec-transactions

Dorks are not working